hudi-common/src/main/java/org/apache/hudi/metadata/HoodieMetadataPayload.java

```diff
@@ -249,7 +249,7 @@ protected HoodieMetadataPayload(String key, int type,
    *
    * @param partitions The list of partitions
    */
-  public static HoodieRecord<HoodieMetadataPayload> createPartitionListRecord(List<String> partitions) {
+  public static HoodieRecord<HoodieMetadataPayload> createPartitionListRecord(Collection<String> partitions) {
```

Comment:
so that we can take in both list and set?

Reply:
Yes